packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md

```diff
@@ -0,0 +1,284 @@
+| | * Display connection settings
+| | * Extend virtual disk
+| | * Modify device settings
+| | * Query Fault Tolerance compatibility
+| | * Query unowned files
+| | * Reload from path
+| | * Remove disk
+| | * Rename
+| | * Reset guest information
+| | * Set annotation
+| | * Toggle fork parent
+| | * Upgrade virtual machine compatibility
+| | Guest operations
+| | * Guest operation alias modification
+| | * Guest operation alias query
+| | * Guest operation modifications
+| | * Guest operation program execution
+| | * Guest operation queries
+| | Interaction
+| | * Power off
+| | * Power on
+| | Inventory
+| | * Create from existing
+| | * Create new
+| | * Move
+| | * Remove
+| | Provisioning
+| | * Allow disk access
+| | * Allow file access
+| | * Allow read-only disk access
+| | * Allow virtual machine download
+| | * Allow virtual machine files upload
+| | * Clone template
+| | * Clone virtual machine
+| | * Create template from virtual machine
+| | * Customize guest
+| | * Deploy template
+| | * Mark as template
+| | * Mark as virtual machine
+| | * Modify customization specification
+| | * Promote disks
+| | * Read customization specifications
+| | Service Configuration
+| | * Allow notifications
+| | * Allow polling of global event notifications
+| | * Manage service configurations
+| | * Modify service configuration
+| | * Query service configurations
+| | * Read service configuration
+| | Snapshot management
+| | * Create snapshot
+| | * Remove snapshot
+| | * Rename snapshot
+| | * Revert to snapshot
+| | vSphere Replication
+| | * Configure replication
+| | * Manage replication
+| | * Monitor replication
+| vApp | Import
+| | View OVF environment
+| | vApp application configuration
+| | vApp instance configuration
+| vSphere Tagging| Create vSphere Tag
+| | Edit vSphere Tag
+
+
+### Optional Settings 
+The requirement configurations given below are optional.
+
+* DNS Mapping
+   - Static IP DNS: [Name servers] Comma separated DNS addresses (e.g: 8.8.8.8, 192.168.0.8), required only for static IP allocation.
+* SMTP Settings
+   - Configure SMTP settings to enable the Spectro Cloud platform to send out email notifications. Email Notifications are sent out to new users when they are on boarded to the platform so they can activate their accounts.
+* Certificate (optional)
+   - Provide the desired SSL or TLS server certificates to support external access to valid HTTPs.
+* FTP Location (backups)
+   - An accessible FTP location to backup data.
+   	
+# Quick Start Mode
+
+The Spectro Cloud On-Prem Quick Start Mode is a single node installation of the Spectro Cloud platform. This deployment is used to quickly understand the capabilities of the Spectro Cloud platform. For production purposes the enterprise version is recommended. The minimum requirements for Quick Start version is outlined below:
+
+### Hardware Requirements 
+
+Hardware requirements will describe the CPU, memory, and disk requirements for the nodes.
```

Comment:
Can mention minimum requirement. And also we can mention that the installer will take care of allocating the resources.